### PR TITLE
Fix eslint errors and warnings

### PR DESCRIPTION
**Pull request by aabdaab**
- split too long lines (more than 140 chars)
- add jest recognition for linter
- fix other issues threw by linter

### CODE REVIEW

File: app/javascript/app/components/about/about-partners/about-partners.js [hunk 1]
```diff
@@ -41,7 +41,8 @@ const partnershipWith = {
         src: climateActionTrackerImage
       },
       description:
-        'joined the Climate Watch development team and provided support in developing concepts for the scope and functions of Climate Watch.'
+        'joined the Climate Watch development team and provided support in developing concepts for the ' +
```
Comment (Bluesmile82) [inline app/javascript/app/components/about/about-partners/about-partners.js]: We prefer to use always template literals than concatenation: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

File: app/javascript/app/data/SEO.js [hunk 2]
```diff
@@ -1,16 +1,23 @@
 export const TITLE = 'Climate Watch: Data for Climate Action';
 export const HOME_PAGE =
-  'Climate Watch is an open online platform designed to empower users with the climate data, visualizations and resources they need to gather insights on national and global progress on climate change, sustainable development, and help advance the goals of the Paris Agreement.';
+  'Climate Watch is an open online platform designed to empower users with the climate data, ' +
```
Comment (Bluesmile82) [inline app/javascript/app/data/SEO.js]: Same, template literals make it easier to update the text

File: app/javascript/app/components/about/about-partners/about-partners.js (hunk 1 above)
Comment (aabdaab) [reply app/javascript/app/components/about/about-partners/about-partners.js]: I don't know to what extent should we listen to eslint ;> when you use template literals eslint throws "Strings must use singlequote". I used concatenation as the line was too long. Should I revert those changes?

File: app/javascript/app/data/SEO.js (hunk 2 above)
Comment (aabdaab) [reply app/javascript/app/data/SEO.js]: same as above, eslint warnings

File: app/javascript/app/components/about/about-partners/about-partners.js (hunk 1 above)
Comment (Bluesmile82) [reply app/javascript/app/components/about/about-partners/about-partners.js]: I will revert the changes in these cases. It's better to use string literals in my opinion. You could also disable the es-lint must use singlequote errors. But I'm not sure if it worths it.

**Pull request merged**